ccpp/suites/suite_FV3_GFS_v16_gfdlmpv3.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@dustinswales See https://github.com/ufs-community/ufs-weather-model/pull/2503#issuecomment-2825011458: you need to split these new SDFs into the new phys_ps and phys_ts groups.

Reply:
Done.